components/ActionSheet/tests/unit/ActionSheetTest.swift

```diff
@@ -78,4 +81,77 @@ class ActionSheetTest: XCTestCase {
     }
     
   }
+
+  func testDefaultColors() {
+    // Given
+    let titleTest = "Title"
+
+    // When
+    actionSheet.title = titleTest
+
+    // Then
+    if let title = getLabel(with: titleTest) {
+      XCTAssertEqual(title.textColor, .black)
+    } else {
+      XCTFail("No title label")
+    }
+
+    // Given
+    let messageTest = "Message"
+
+    // When
+    actionSheet.message = messageTest
+
+    // Then
+    if let message = getLabel(with: messageTest) {
+      XCTAssertEqual(message.textColor, .black)
```

Comment:
This seems to be missing an assert for the title text color once a `message` value is assigned.